iXBRLViewerPlugin.nuspec

```
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>iXBRLViewerPlugin</id>
+    <version>$version$</version>
+    <authors>mms solutions ag</authors>
+    <description>ixbrl viewer plugin package for deployment</description>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <copyright>Copyright (c) mms solutions ag</copyright>
+    <repository type="git" url="https://github.com/mmssolutionsio/taggingplus" />
```

I think here we should set the repository of the viewer fork as the viewer is independent from taggingplus.
```suggestion
    <repository type="git" url="https://github.com/mmssolutionsio/ixbrl-viewer" />
```

---

Good point. @iappert ?

---

yes